pip_audit/service/service.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC
+from dataclasses import dataclass
+from typing import Dict, List, Optional
+
+from packaging.version import Version
+
+
+@dataclass(frozen=True)
+class Dependency:
+    package: str
```

Comment:
Maybe overkill, but we might want to consider doing a `typing.NewType` for the `package` field in the future -- we expect package names to be consistent with PEP 423 (?), so having some sort of type that reflects that would probably be good.